cmd/contrib/main.go

```diff
@@ -191,18 +191,7 @@ func newPlugin() error {
 	// "GitHub API Key" => "API Key"
 	// "Credentials" => "Credentials"
```

Comment:
This comment seems mostly duplicated from the godoc of the function. Do we need both?

Reply:
I propose we maintain both examples (especially the second one) because we have a check in place to use the last word always, irrespective of whether that word is greater than seven characters or not.